VideoWeb/VideoWeb/ClientApp/src/app/services/participant-status-update.service.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { Router } from '@angular/router';
+import { ParticipantStatusUpdateService } from 'src/app/services/participant-status-update.service';
+import { VideoWebService } from 'src/app/services/api/video-web.service';
+import { MockLogger } from 'src/app/testing/mocks/MockLogger';
+import { Logger } from 'src/app/services/logging/logger-base';
+import { TestBed, fakeAsync } from '@angular/core/testing';
+
+class MockRouter {
+    public url = '/check-equipment/1234-1234-1234';
+}
+
+describe('ParticipantStatusUpdateService', () => {
+
+    let videoWebServiceSpy: jasmine.SpyObj<VideoWebService>;
```

Comment:
Could you remove the test bed here and follow suite like the other tests. Thanks

Reply:
done

Reply:
done